exercises/rotational-cipher/src/example/java/RotationalCipher.java

```diff
@@ -0,0 +1,29 @@
+public class RotationalCipher {
+    public String rotate(String data, int  shiftKey){
```

Comment:
It's best practice to leave a space before `{`. There are a few places where you've left out whitespaces. To easily fix formatting issues like this you can use cmd + alt + L if you're using intellij (on a mac, it's ctrl + alt + L on Windows). You can also [do the same thing in eclipse](http://stackoverflow.com/questions/15655126/how-to-auto-format-code-in-eclipse) :)